statistics/handle.go

```diff
@@ -56,22 +56,16 @@ type Handle struct {
 	feedback []*QueryFeedback
 
 	Lease time.Duration
-	// loadMetaCh is a channel to notify a load stats operation has done.
-	loadMetaCh chan *LoadMeta
 }
 
 // Clear the statsCache, only for test.
 func (h *Handle) Clear() {
```

Comment:
should we acquire the `h.mu.Mutex` here?

Reply:
No need to because it is only used for the test, and stats test won't run concurrently.